src/backend/main.ts

```diff
@@ -944,6 +945,32 @@ ipcMain.handle(
       mainWindow?.hide()
     }
 
+    if (autoCloseLauncher) {
+      mainWindow?.hide()
+      const timeoutId = setTimeout(() => {
+        try {
+          mainWindow?.close()
+        } catch (error) {
+          logError(
+            'Error closing the Heroic window after launch:',
+            LogPrefix.Backend
+          )
+          mainWindow?.show()
```

Comment:
what could be the error here? I'm confused why on error you are re-showing the window (which maybe got hidden by the `minimize heroic to the sidebar` feature for example)

I don't think this feature should mess with hiding/showing the main window, it should only try to close heroic and let the hiding/showing control to the other feature that's specific for it